LiveSched/src/test/java/dev/coms4156/project/livesched/ScheduleUnitTests.java

```diff
@@ -23,145 +25,134 @@
 @ContextConfiguration
 public class ScheduleUnitTests {
 
-  private Schedule testSchedule;
-  private String scheduleId = "TestSchedule";
-  private List<ResourceType> resourceTypes;
-  private double maxDistance = 10.0;
-  private Task testTask;
-  private ResourceType testResourceType;
+  private List<Task> mockTasks;
+  private ResourceType mockResourceType;
+  private Task mockTask1;
+  private Task mockTask2;
+  private Resource mockResource1;
+  private Resource mockResource2;
+  private Location taskLocation;
+  private Location resourceLocation;
+
+  private final String testScheduleId = "Schedule ID1";
+  private final double maxDistance = 100.0;
+  // private final double longitude = -73.96;
 
   /**
    * Set up to be run before all tests.
    */
   @BeforeEach
   void setupScheduleForTesting() {
-    resourceTypes = new ArrayList<>();
-    testResourceType = new ResourceType("Hospital", 5, 40.7128, -74.0060);
-    resourceTypes.add(testResourceType);
-    
-    testSchedule = new Schedule(scheduleId, resourceTypes, maxDistance);
-    
-    Map<ResourceType, Integer> resourceList = new HashMap<>();
-    resourceList.put(testResourceType, 1);
-    LocalDateTime startTime = LocalDateTime.now().plusHours(1);
-    LocalDateTime endTime = startTime.plusHours(2);
-    testTask = new Task("TestTask", resourceList, 3, startTime, endTime, 40.7128, -74.0060);
+    mockTask1 = mock(Task.class);
```

Comment:
Love how you mocked the behaviors of the class methods for simplicity :)

Reply:
Yes, will do integration tests to check